pandera/api/polars/types.py

```diff
@@ -20,7 +20,8 @@ class CheckResult(NamedTuple):
 
 
 PolarsCheckObjects = Union[pl.LazyFrame, pl.DataFrame]
-
+PolarsFrame = TypeVar("PolarsFrame", pl.LazyFrame, pl.DataFrame)
+PolarsFrame2 = TypeVar("PolarsFrame2", pl.LazyFrame, pl.DataFrame)
```

Comment:
what's `PolarsFrame2` for?

Reply:
I had it in this:
```python
def _to_frame_kind(obj: PolarsFrame, kind: type[PolarsFrame2]) -> PolarsFrame2:
```
to indicate that the first and second arguments don't need to be in sync. But codecov is not happy about this either, obj is always a lazyframe right now, so I might just remove it (and in fact I think it could just be a union in this case as it doesn't affect the return type).